dexs/OhMyCrypt/index.ts

```diff
@@ -0,0 +1,96 @@
+import { Adapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import type { ChainEndpoints, SimpleAdapter } from "../../adapters/types";
+import type { Chain } from "@defillama/sdk/build/general";
+import { request, gql } from "graphql-request";
+import { getTimestampAtStartOfDayUTC } from '../../utils/date';
+
+// Subgraphs endpoints
+const endpoints: ChainEndpoints = {
+  [CHAIN.BASE]: "https://api.thegraph.com/subgraphs/name/ohmycrypt/omcoff",
+};
+
+function formatEthers(weiAmount:string, decimals = 18) {
+    const etherValue = parseFloat(weiAmount) / 10 ** decimals;
+    return etherValue.toString() + ' Ether';
+}
+
+// Fetch function to query the subgraphs
+const graphs = (graphUrls: ChainEndpoints) => {
+    return (chain: Chain) => {
+      return async (timestamp: number) => {
+        const todaysTimestamp = getTimestampAtStartOfDayUTC(timestamp);
+        const graphQueryusdcday = gql`
+          query {
+            activeOrders(where: {timestamp_gte: ${todaysTimestamp}, t2:"0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913" }) {
+              t2a
+            }
+          }
+          `;
+          const graphQueryusdcalltime = gql`
+          query {
+            activeOrders(where: {t2:"0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913"}) {
+              t2a
+            }
+          }
+          `;
+           const graphQuerydaiday = gql`
+           query {
+             activeOrders(where: {timestamp_gte: ${todaysTimestamp}, t2:"0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb" }) {
```

Comment:
could you change query like this https://github.com/DefiLlama/dimension-adapters/blob/master/fees/curve.ts#L32
fiter yesterday data